NetworkMonitor/Classes/Debug/Controller/FNMDebugListingViewController.swift

```diff
@@ -280,8 +284,10 @@ private extension FNMDebugListingViewController {
         typealias RecordFilter = (FNMHTTPRequestRecord) -> Bool
 
         let searchFilter: RecordFilter = {
-            guard let currentSearchFilter = self.currentSearchFilter() else { return true }
-            return $0.request.url?.absoluteString.lowercased().contains(currentSearchFilter.lowercased()) == true
+            guard let currentSearchFilters = self.currentSearchFilters(),
+                  let requestUrl = $0.request.url?.absoluteString.lowercased() else { return true }
+
+            return currentSearchFilters.map { $0.lowercased() } .contains(where: requestUrl.contains)
```

Comment:
`map` and `contains`, you're iterating twice. 

Suggestion:

```swift
            return currentSearchFilters.contains(where: { requestUrl.contains { $0.lowercaased() } })
```

Reply:
That won't compile, since you need to return a `Bool` on the second `contains` closure block.

Reply:
Oh.. I think this will.           

```swift 
return currentSearchFilters.contains(where: { requestUrl.contains($0.lowercased()) })
```

Reply:
Done 👍